src/envs/playroom.py

```diff
@@ -460,22 +560,140 @@ def _PutOnTable_policy(self, state: State, memory: Dict,
         arr = np.clip(arr, self.action_space.low, self.action_space.high)
         return Action(arr)
 
+    def _Move_policy(self, state: State, memory: Dict,
+                      objects: Sequence[Object], params: Array) -> Action:
+        del memory  # unused
+        robot, = objects
+        fingers = state.get(robot, "fingers")
+        arr = np.r_[params[:-1], 1.0, params[-1], fingers].astype(np.float32)
+        arr = np.clip(arr, self.action_space.low, self.action_space.high)
+        return Action(arr)
+
+    @staticmethod
+    def _Move_initiable(state: State, memory: Dict, objects: Sequence[Object],
+                        params: Array) -> bool:
+        del state, memory, objects, params  # unused
+        return True  # can be run from anywhere
+
+    @staticmethod
+    def _Move_terminal(state: State, memory: Dict, objects: Sequence[Object],
+                       params: Array) -> bool:
+        del state, memory, objects, params  # unused
+        return True  # always 1 timestep
+
+    def _OpenDoor_policy(self, state: State, memory: Dict,
+                         objects: Sequence[Object], params: Array) -> Action:
+        del memory  # unused
+        door, = objects
+        door_pose = np.array([state.get(door, "pose_x"),
+                              state.get(door, "pose_y"),
+                              self.door_button_z])
+        arr = np.r_[door_pose+params[:-1], params[-1], 1.0].astype(np.float32)
+        arr = np.clip(arr, self.action_space.low, self.action_space.high)
+        return Action(arr)
```

Comment:
if the opendoor and closedoor options are indeed the same, I recommend just writing the code once (call it `ToggleDoor` or something) and use the same init/policy/term in constructing the two different ParameterizedOptions.